knowledge_base/_Glossary.md

```diff
@@ -0,0 +1,250 @@
+
+Glossary terms referenced in a given term's definition are designated with CAPITAL letters. Relevant models and related entries may also be referenced; see [Subscription](#subscription) for an example.
+
+## Contents
+
+- [Chain Concepts](#chain-concepts)
+  * [Chain Entity](#chain-entity)
+  * [Chain Node](#chain-node)
+  * [Delegate Contract](#delegate-contract)
+  * [Module](#module)
+- [Common App](#common-app)
+  * [Bridge](#bridge)
+  * [Comment](#comment)
+  * [Community](#community)
+  * [Core](#core)
+  * [Forum](#forum)
+  * [Group](#group)
+  * [Link](#link)
+  * [Member](#member)
+  * [Notification](#notification)
+  * [Page Scope](#page-scope)
+  * [Profile](#profile)
+  * [Reaction](#reaction)
+  * [Poll](#poll)
+  * [Role](#role)
+  * [Sessions](#sessions)
+  * [Sign-In](#sign-in)
+  * [Stake](#stake)
+  * [Subscription](#subscription)
+  * [Template](#template)
+  * [Thread](#thread)
+  * [Topic](#topic)
+  * [User](#user)
+  * [Wallet](#wallet)
+  * [Votes](#votes)
+- [Hexagonal Architecture](#hexagonal-architecture)
+  * [Port](#port)
+  * [Adapter](#adapter)
+- [Praxis](#praxis)
+  * [Blocked](#blocked)
+  * [Fast-Follow](#fast-follow)
+  * [Issues, Tickets, & Stories](#issues-tickets--stories)
+- [Change Log](#change-log)
+
+## Chain Concepts
+
+### Chain Base
+
+An architectural layer of blockchain atop which NETWORK protocols are built.
+
+### Chain Entity
+
+A Chain Entity is a backend object, stored in the ChainEntities table, which consists of chain event bundles and values received from a smart contract.
+
+### Chain Node
+
+An endpoint, referring to external RPC servers, used to fetch updates and query data from blockchains.
+
+### Delegate Contract
+
+A delegate contract is a smart contract that contains the functions, events, and all business logic for a proxy contract.
+
+### Module
+
+Some CHAINS use modules rather than contracts. Modules are used predominantly within the Cosmos and Substrate ecosystems.
+
+### Network
+
+An ecosystem of CHAIN NODES, or blockchain, built atop a CHAIN BASE.
+
+## Common App
+
+### Bridge
+
+The Product Team uses "bridge" and "bridging" language, in user-facing comms, to refer to implementation-agnostic, cross-platform integrations such as [Discobot](./Discobot.md) and Farcaster.
+
+### Comment
+
+A short Markdown text object responding to a THREAD, which can be REACTED to.
+
+See [models/comment.ts](../libs/model/src/models/comment.ts).
+
+### Community
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+A community is a space on Common, joinable by PROFILES, that owns metadata and is defined by a set of ROLES and PERMISSIONS for participation.
+
+See [models/community.ts](../libs/model/src/models/community.ts)
+
+### Core
+
+A set of features considered central to Common, which interface out with first-party apps. As of 240123, the core consists of a SIGN-IN system, a NOTIFICATION system, and a GROUP system.
```

Comment:
Where was this info from re "Core"?

Reply:
I believe this came from a hybridization of a 2023 Notion Glossary concept, and informal discussions with team members re: Common within a hexagonal architecture framework.

Does this conceptualization not seem up-to-date?

Reply:
I just don't hear the term used regularly.